go.mod

```diff
@@ -133,7 +133,7 @@ require (
 	k8s.io/sample-apiserver v0.0.0
 	k8s.io/system-validators v1.5.0
 	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
-	sigs.k8s.io/structured-merge-diff/v4 v4.1.2
+	sigs.k8s.io/structured-merge-diff/v4 v4.1.3-0.20211027235903-b4c53cbfc9dc
```

Comment:
normally we tag releases upstream before updating, to avoid pulling in intermediate untagged versions like this. is there a reason to pull in an unreleased commit?

Reply:
We might pull-in additional changes on that repo before we tag/release. We'll probably tag no matter what before code-freeze.

Reply:
was this ordering fix going to target backport as well?

Reply:
We're discussing that, I think it'd be ideal. I think the backport are all on the same major branch of smd so maybe we should tag in-between.

Reply:
Okay let's defer that until we make a tag with `sigs.k8s.io/structured-merge-diff/v4`.

Reply:
I tagged `v4.2.0`: https://github.com/kubernetes-sigs/structured-merge-diff/tree/v4.2.0